test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/IntegerGeneratorTest.java

```diff
@@ -0,0 +1,218 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class IntegerGeneratorTest {
```

Comment:
Not sure if we are adding any value by having lots of test cases which are only testing different ranges? I will leave someone else (Allan or Matt) to decide on this.